Go sample application using Azure Log Analytics data collector API

saleem-mirza opened the pull request:
Add sample application in Go language  using REST api
Add syntax highlighting for sample scripts for PowerShell, C#, Python and Go

azurecla commented:
Hi __@v-anpasi__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

saleem-mirza committed:
Add Golang sample application

Add sample application in Go language  using REST api
Add syntax highlighting for sample scripts for PowerShell, C#, Python and Go
